corehq/apps/cloudcare/static/cloudcare/js/form_entry/spec/entries_spec.js

```diff
@@ -34,7 +35,7 @@ describe('Entries', function () {
         };
         spy = sinon.spy();
         $.subscribe('formplayer.' + Const.ANSWER, spy);
-        this.clock = sinon.useFakeTimers();
+        this.clock = sinon.useFakeTimers(new Date("2020-03-15 15:41").getTime());
```

Comment:
this would've been a fun test time bomb to start triggering in 6 years

Reply:
💣